includes/irrKlang/irrKlang.h

```diff
@@ -118,7 +118,7 @@
  *	<ul>
  *       <li>ALSA</li>
  *	</ul>
- *  <li>MacOSX (x86 and PPC)</li>
+ *  <li>Mac OS X (x86 and PPC)</li>
```

Comment:
It's possible certain files come from upstreams. I can try to queue sending PRs to them as well...

Note that the modern platform is `macOS` not `Mac OS X`, but it runs on x86_64 and arm...